test/plugins/_index.js

```diff
@@ -36,7 +36,15 @@ describe('plugins tests', function() {
                         plugins = {},
                         svgo;
 
-                    plugins[name] = (params) ? JSON.parse(params) : true;
+                    var getParams = function() {
+                        var t = {};
+                        /* jshint ignore:start */
+                        eval('t = ' + params);
```

Comment:
Why `eval`? For regular expressions you can use `RegExp` constructor. `eval` is way too fragile and not-secure.

Reply:
Json.parse know about RegExp?

Reply:
FYI, JSON.parse [has optional `reviver` parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse) that allow custom parsing.

Reply:
> parsing is transformed before being returned
Transformed, not parsed

Reply:
```
JSON.parse('{ "d": /^M0\s0h\d0v\d0H0z$/i }')
JSON.parse('{ "d": /^M0\s0h\d0v\d0H0z$/i }', (k, v) => { /* ... */ })
```

are equal.

Do you really think, that parsing string that looks like regexp will be better instead of 3 lines of code?